File: AirlineServiceAPI/Models/Passenger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AirlineService.Data {
    public class Passenger {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string Job { get; set; } = null!;
        public string Email { get; set; } = null!;
        public int Age { get; set; }
        //public ICollection<Booking>? Bookings { get; set; }
    }
}

File: src/dataModels/Passenger.ts
interface Passenger {
    id: number;
    firstName: string;
    lastName: string;
    job: string;
    email: string;
    age: number;
}
export type AddPassengerDTO = {
    firstName : string,
    lastName : string,
    job : string,
    email : string,
    age : number,
}

export default Passenger;